en_US/deploy/install-docker.md

```diff
@@ -4,11 +4,17 @@ This chapter introduces how to use the official Docker image to install and run
 
 :::tip Note
 
-1. If you want to keep the data, mount the EMQX data directory (`/opt/emqx/data`) out of the container, so that the data will persist even if the container no longer exists.
+1. If you want to persist the EMQX Docker container, you need to keep the following directories, so that the data will persist even if the container no longer exists.
```

Comment:
```suggestion
1. If you want to persist data generated in the EMQX Docker container, you need to keep the following directories, so that the data will persist even if the container no longer exists.
```